# ArduinoDriversMsg.h

DepthSensorMsg depth_sensor
ThrusterArrayMsg thruster_array
